book/intro.md

```diff
@@ -1,12 +1,55 @@
-# Welcome to AI Singapore's AI Practitioner Handbook
-
 ```{image} ./assets/images/logos/aisg-ai-practitioner-handbook-banner.png
 :alt: AISG AI Practitioner Handbook Banner
 :class: bg-primary mb-1
 :align: center
 ```
+# Welcome to AI Singapore's AI Practitioner Handbook
 
-Overview of the book here.
+Contributor: Laurence Liew, Director (AI Innovation)
 
-```{tableofcontents}
-```
+---
+
+## Introduction
+
+This handbook is an accumulation of AI Singapore’s Innovation and Platforms Engineering team's experience in delivering more than X??? AI Minimum Viable products (MVP) under the [100E](https://www.aisingapore.org/innovation/100e) programme over the last 5 years. 
```

Comment:
@kevinosl Could I check if you have the latest numbers for completed 100Es?

Reply:
The whole 100E program has 86. But those that our team delivered is 40+, so you can use X=40